changelog/0007-reorganize-pillows.yml

```diff
@@ -0,0 +1,107 @@
+title: Reorganize pillows
+key: reorganize-pillows
+date: 2018-11-26
+
+optional_per_env: no
+
+min_commcare_version: asdasdsd # to be filled after the main PR gets merged
```

Comment:
Will need to add the merge commit of this PR https://github.com/dimagi/commcare-hq/pull/21329/